modules/nf-scil/tracking/localtracking/main.nf

```diff
@@ -0,0 +1,87 @@
+process TRACKING_LOCALTRACKING {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'biocontainers/scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(fodf), path(tracking_mask), path(seed)
+
```

Comment:
Modify input to include PVE maps (wm, gm, csf) and compute tracking_mask and seeding_map in module.

tuple val(meta), path(wm), path(gm), path(csf), path(fodf), path(fa)